libsplinter/src/mesh/mod.rs

```diff
@@ -558,4 +582,65 @@ mod tests {
         let tls = create_test_tls_transport(true);
         test_add_remove_connections(tls, "127.0.0.1:0");
     }
+
+    #[test]
+    // Test mesh can be shutdown
```

Comment:
Need more descriptive test comments - see PBFT tests for examples.

Reply:
updated